config/application.rb

```diff
@@ -29,5 +29,8 @@ class Application < Rails::Application
 
     # since web console usage depends on subdomain rules
     config.web_console.development_only = false
+
+    # generates remote forms with form_with 
+    config.action_view.form_with_generates_remote_forms = true
```

Comment:
@Pralish you mentioned that the fix was to:

    Either add 'local: false' to the form
    OR update the config [Implemented]

Can we only add the local flag? I feel like setting the config could block us from Rails upgrades down the road

Reply:
@donrestarone , We will have to add local flag to comfertable_mexican_sofa repo. Is that ok?

Reply:
@Pralish yeah lets do that and add a test for the redirect!

Reply:
no hurry, take your time on the change since we have a queue to drain @Pralish